apiv2/src/admin/infra/sejours/cle/classe/guard/ClasseAdminCle.guard.ts

```diff
@@ -30,7 +30,7 @@ export class ClasseAdminCleGuard implements CanActivate {
                 return false;
             }
             return (
-                etablissement.referentEtablissementIds.includes(request.user.id) &&
+                etablissement.referentEtablissementIds.includes(request.user.id) ||
```

Comment:
avec le AND, ça veux dire qu'un chef d'établissement n'est pas un admin clé de la classe, cela semble incorrect 🤔 
on n'est normalement pas chef d'établissement et coordinateur en même temps.